src/Jackett.Common/Indexers/CardigannIndexer.cs

```diff
@@ -1603,11 +1603,30 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                                             if (query.ImdbID != null && TorznabCaps.SupportsImdbMovieSearch)
                                                 break; // skip andmatch filter for imdb searches
 
-                                            if (!query.MatchQueryStringAND(release.Title, CharacterLimit))
```

Comment:
Instead of duplicating functionality of `MatchQueryStringAND` here with the `.Keywords` variable, it looks like the better option would be to just pass the `.Keywords` variable into `MatchQueryStringAND` as the third parameter `queryStringOverride`.

Maybe we should precheck for `string.IsNullOrWhitespace` first though and default to the original query title. The resulting code would look like this:
```C#
var queryKeywords = (string)variables[".Keywords"];
if (string.IsNullOrWhiteSpace(queryKeywords))
    queryKeywords = null;
if (!query.MatchQueryStringAND(release.Title, CharacterLimit, queryKeywords))
```
Everything else should remain how it was, I believe. Please test this to make sure.